std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -1045,6 +1047,28 @@ struct SharedFreeList(ParentAllocator,
         resetNodes();
         return result;
     }
+
+    /**
+    Nonstandard function that minimizes the memory usage of the freelist by
+    freeing each element in turn. Defined only if $(D ParentAllocator) defines
+    $(D deallocate).
+    */
+    static if (hasMember!(ParentAllocator, "deallocate") && !unchecked)
+    void minimize() shared
+    {
+        lock.lock();
+        scope(exit) lock.unlock();
```

Comment:
We could add `minimize(size_t quantum = size_t.max)` meaning "minimize quantum bytes at most". But I'd say let's accumulate more experience with the unbounded minimize.